.github/workflows/test.yaml

```diff
@@ -14,6 +14,7 @@ jobs:
   test:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
What was wrong with fail fast?

Reply:
When fail-fast was set to true first failed action cancelled all the others.